components/CardInfo.js

```diff
@@ -0,0 +1,24 @@
+import { CARDS } from '../shared/cards';
```

Comment:
What do you think about switching the names around? So calling the data `CARD_INFO` and then calling the component `CARDS`.